app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Admin
+  class CategoriesController < Admin::BaseController
+    before_action :set_category, only: %i[show edit destroy update]
+
+    def index
+      @pagy, @category = pagy(Category.all, items: 5)
+      respond_to do |format|
+        format.html
+        format.csv { send_data Category.all.to_csv, filename: "categories-#{Date.today}.csv" }
```

Comment:
Can we just do `Category.to_csv`?

Reply:
farzam bhai we cant do that because due to pagination it will only populate max 5 items in the CSV file and we don't want that